src/components/FileInput.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { ReactElement } from "react";
+import { textSans } from "@guardian/src-foundations/typography";
+import { css } from "@emotion/core";
+import { neutral } from "@guardian/src-foundations";
+
+interface FileInputProps {
+    required: boolean;
+    name: string;
+    label: string;
+}
+
+const optionalLabelStyles = css`
+    ${textSans.small({ fontStyle: "italic" })};
+    color: ${neutral[46]};
+`;
+
+const labelStyles = css`
+    ${textSans.medium({ fontWeight: 'bold' })};
+`;
+
+const optionalLabel = (required: boolean): ReactElement =>
+    required ? <></> : <span css={optionalLabelStyles}>Optional</span>
```

Comment:
Is there a performance reason for this? or is it for readability?

Reply:
The [recommendation from the docs](https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering) to prevent a something from rendering is to return `null`. One possible difference is that `Fragment` may create an object in the tree that React builds up - so there might be a performance difference.